audit_config.yaml

```diff
@@ -16,7 +16,14 @@ S3_PUBLIC_POLICY_GETOBJECT_ONLY:
   severity: Info
   is_global: True
   group: S3
-
+  
+S3_ENCRYPTION:
+  title: s3 encryption is not enabled
+  description: encryption is not done on these buckets
+  severity: Medium
```

Comment:
I would set this as Low severity, as the threat is generally just if someone breaks into an AWS facility and steals the hard-drive. There are additional security benefits if KMS encryption is used with a custom key, but because this only checks if the bucket is encrypted generally, I wouldn't set the severity as high.